src/components/labelled-date-input/LabelledDateInput.css

```diff
@@ -0,0 +1,12 @@
+.form-control {
+  width: 150px;
+  padding: 5px;
+  border-radius: 4px;
+  border: 1px solid black;
+  margin-left: 10px;
+  text-align: center;
+}
+
+.rdtPicker {
```

Comment:
it's not good practice to override some lib styles this way. But without it picker looks a bit weird in the current design and there is no way we can change it via component api. So I'm ok with such overriding here.